inheritance/src/main/java/inheritance/Library.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package inheritance;
+
+public class Library {
+    public boolean someLibraryMethod() {
+
+        return true;
+    }
+
+    // Restaurant class
+    public static class Restaurant {
+       static String name;
```

Comment:
You don't want this to be static. By making it static, each individual instance of a restaurant you create won't have a `name`. Instead, the `name` variable will belong to the class.